docs/usage.md

```diff
@@ -942,13 +983,17 @@ Specify a SNP panel in [EIGENSTRAT](https://github.com/DReichLab/EIG/tree/master
 
 Specify calling method to use. Options: randomHaploid, randomDiploid, majorityCall. Default: `'randomHaploid'`
 
+#### `--pileupcaller_transitions_mode`
+
+Specify if genotypes of transition SNPs should be called, set to missing, or excluded from the genotypes respectively. Options: AllSites, TransitionsMissing, SkipTransitions. Default: '`AllSites`'
```

Comment:
```suggestion
Specify if genotypes of transition SNPs should be called, set to missing, or excluded from the genotypes respectively. This only applies to double-stranded libraries, as single-stranded libraries are genotyped in a way that eliminates false transitions being called because of DNA damage. Options: AllSites, TransitionsMissing, SkipTransitions. Default: '`AllSites`'
```

Reply:
Check for more of these quote and backtick flips just in case. I found some, but may have missed a few more.